src/assign-graders/utils.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-ignore
+import yaml from "js-yaml";
+import fs from "fs";
+import { homedir } from "os";
+import { join } from "path";
+import { Config, Grader } from "./types";
+
+/**
+ * Load list of graders from config file
+ */
+export function loadConfig(): Config {
+    const filePath = join(homedir(), "ght-graders.yml");
```

Comment:
this won't work as a snap because of the confinement that the snap is in.
I will push a fix for this in my commit.

Reply:
Good, that will save us some headache. But now I'm confused: isn't it the same that we're using here? https://github.com/canonical/ght/blob/main/src/common/constants.ts#L4

Reply:
that one is autogenerated by the app and only used by the app, so it's an internal secret file, the user don't need to access it at all, so it's inside the Snap's Home directory which is different that the User's usual home directory and is only accessible by the snap app GHT.

Reply:
Cool! TIL